tests/contrib/utils/test_weekday.py

```diff
@@ -37,6 +37,6 @@ def test_weekday_name_value(self):
             self.assertEqual(e.name, weekday)
             self.assertTrue(e in WeekDay)
             self.assertTrue(0 < e < 8)
-            self.assertTrue(type(e) is WeekDay)
+            self.assertTrue(isinstance(e, WeekDay))
```

Comment:
While we are here making changes could you correct these to:

             self.assertIsIinstance(e, WeekDay)

etc

Reply:
@ashb Done!